syntax="proto3";

package chat_completion;

service ChatService {
    rpc ask (ChatMessage) returns (ChatMessage) {}
    rpc chat (stream ChatMessage) returns (stream ChatMessage) {}
    rpc write_article_by_title (ChatMessage) returns (stream ChatMessage) {}
    rpc transcribe_judge (ChatMessage) returns (ChatMessage) {}
};

message ChatMessage {
    uint64 logid = 1;
    uint64 uid = 2;
    string content = 3;
};

